game-core/src/main/java/swinglib/JButtonBuilder.java

```diff
@@ -24,7 +24,7 @@
   private String title;
   private String toolTip;
   private Runnable actionListener;
-  private boolean visible = true;
+  private final boolean visible = true;
```

Comment:
This should be removed, JComponents are visible by default.

Reply:
Agree, should finish removing it while started; I finished the removal in: e193035